Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,31 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+
+            if (array == null)
+            {
+                return;
+            }
+
+            var flag = true;
```

Comment:
Maybe you can find a better name for that variable, which is more descriptive.
It is fine for me but maybe something like hasSwapped would make more sense to the reader in general.

Reply:
Also logical variable are prefixed with "is".

Reply:
done. by removing it and replaced it with a break